pkgs/development/tools/build-managers/gradle/default.nix

```diff
@@ -15,7 +16,6 @@ let
       symlinkJoin,
       concatTextFile,
       makeSetupHook,
-      nix-update-script,
 
       # This is the "current" version of gradle in nixpkgs.
       # Used to define the update script.
```

Comment:
Can we get rid of this now? (Edit: Just noticed the merge queued, I'll do it in my override cleanup PR. Not necessary for this one.)

Reply:
Already done :)